library/objective-c/EnvoyEngineImpl.m

```diff
@@ -430,6 +430,119 @@ static void ios_track_event(envoy_map map, const void *context) {
   }
 }
 
+#include <CoreFoundation/CoreFoundation.h>
+#include <CoreFoundation/CFArray.h>
+#include "library/common/extensions/cert_validator/platform_bridge/c_types.h"
+#include "openssl/ssl.h"
+
+// Returns a new CFMutableArrayRef containing a series of SecPolicyRefs to be
+// added to a SecTrustRef used to validate a certificate for an SSL server,
+// or NULL on failure.
+CFMutableArrayRef CreateTrustPolicies() {
```

Comment:
Since these are just C APIs, why not move them a layer down into the `library/common` layer so they can be used directly from interfaces that don't delegate to the ObjC implementation, like the C++ builder?